modules/nw-using-service-external-ip.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * ingress/getting-traffic-cluster.adoc
+
+[id="nw-service-external-ip_{context}"]
+= Using a service external IP to get traffic into the cluster
+
+One method to expose a service is to assign an external IP address directly to
+the service you want to make accessible from outside the cluster.
+
+The external IP address that you use must be provisioned on your infrastructure
+platform and attached to a cluster node.
+
+By setting an external IP on the service, {product-title} sets up sets up NAT
+rules to allow traffic arriving at any cluster node to which that IP address is
+attached to be sent to one of the internal pods. This is similar to the internal
```

Comment:
This sentence seems awkward to me. Maybe something like:
_With an external IP on the service, {product-title} sets up sets up NAT
rules to allow traffic arriving at any cluster node attached toi that IP address to be sent to one of the internal pods._